tests/accessibility/.pa11y-ci.js

```diff
@@ -1,23 +1,54 @@
-const { goToCreateNhsAppTemplatePage } = require('./actions/create-nhs-app-template.action');
+const { performCheck, setupBrowser, setupPage } = require('./helpers');
+const {
+  chooseATemplatePage,
+  chooseATemplatePageError,
+  createNHSAppTemplatePage,
+  createNHSAppTemplateErrorPage,
+  reviewNHSAppTemplatePage,
+  reviewNHSAppTemplateErrorPage,
+} = require('./actions');
 
-module.exports = {
-  defaults: {
-    reporters: [
-      'cli', // <-- this is the default reporter
-      [
-        'pa11y-ci-reporter-html',
-        {
-          destination: './.reports/accessibility',
-          includeZeroIssues: true
-        }
+async function pa11yConfig() {
+  const baseUrl = process.env.BASE_URL ?? 'http://localhost:3000';
+  const browser = await setupBrowser();
```

Comment:
So I've done this because of the following reasons:

1. Providing a header like; `Authorization: Basic <value>` successfully authentications and loads the page. However, it does not load up the resources the page fetches. The fetched resources returns a 401. 
2. Providing basic auth credentials in the URL like `https://hello:world@app.com`. Works and loads the page and subsequent resources. However, node `fetch` (which next uses) disallows fetching with credentials in the URL. 
3. I've tried various `chromeLaunch:args` to disable javascript however, this fails either not disabling js. Or when it is disabled then puppeteer fails. 

I'd prefer not todo this and any better ideas woudl be great. One suggestion would be to have have put all resources and static assets not behind basic auth.